userspace/libscap/scap_procs.c

```diff
@@ -310,9 +310,10 @@ int32_t scap_proc_fill_cgroups(struct scap_threadinfo* tinfo, const char* procdi
 		char* token;
 		char* subsys_list;
 		char* cgroup;
+		char* scratch = line;
```

Comment:
No good reason for that. You're right that it's not going to change anything, but let me fix it.